content/applications/services/helpdesk/advanced/close_tickets.rst

```diff
@@ -1,12 +1,12 @@
-===============
-Closing tickets
-===============
+=============
+Close tickets
+=============
 
 Once work has been completed on a *Helpdesk* ticket in Odoo, there are several ways it can be
-closed. Manually closing solved tickets keeps the pipeline up to date, while automatically closing
-inactive tickets prevents unnecessary blocking issues. Allowing customers to close their own tickets
-minimizes confusion around whether an issue is considered solved or not. This results in increased
-operational capacity for support teams, and higher customer satisfaction.
+closed. In addition to manually closing solved tickets, automatically closing inactive tickets keeps
+the pipeline up to date. At the same time, allowing customers to close their own tickets minimizes
+confusion around whether an issue is considered solved or not. This results in increased operational
+capacity for support teams, and higher customer satisfaction.
```

Comment:
```suggestion
closed.

In addition to manually closing solved tickets, automatically closing inactive tickets keeps the
pipeline up-to-date. At the same time, allowing customers to close their own tickets minimizes
confusion around whether an issue is considered solved or not. This results in increased operational
capacity for support teams, and higher customer satisfaction.
```
OPTIONAL: broke up an otherwise slightly "chunky" paragraph